src/main/target/common_fc_pre.h

```diff
@@ -174,3 +174,6 @@
 // #define USE_GYRO_FAST_KALMAN
 #define USE_GYRO_BIQUAD_RC_FIR2
 #endif
+#if (FLASH_SIZE > 512)
```

Comment:
This will cause this to be not built for F7 targets. Are you sure this cannot fit into targets with `(FLASH_SIZE > 256)`?

Reply:
@mikeller linking has failed on some more targets if I have used FLASH_SIZE > 256. I will exclude those targets by undefining this flag only for affected targets - same as I did for Cleanflight.
It looks that resulting binary is much smaller for Cleanflight.

Reply:
What are the targets that are failing? There are some changes in the works that are designed to reduce the flash space usage - once these have been merged there should hopefully be more space for this.

Cleanflight is smaller because new additions to Betaflight have not been merged into cleanflight in a couple of months.

Reply:
@mikeller It was failing for BETAFLIGHTF3 - I have added same undef for this target.

Reply:
Ah. BETAFLIGHTF3 (practically speaking all F3s we support) is a 256 KB target, so `#if (FLASH_SIZE > 256)` will already exclude it.